lua/autorun/lua_reloadent.lua

```diff
@@ -221,7 +221,7 @@ if SERVER then
 	---------------------- Register server-side lua_reloadent ----------------------
 	-- The client is supposed to register less ugly proxy commands for these, which support auto-completition
 	concommand.Add("_lua_reloadentity_sv", function(ply,command,args)
-		if not ply:IsSuperAdmin() then return end
+		if ply:IsValid() and not ply:IsSuperAdmin() then return end

```

Comment:
If the player is invalid, whoever issued the command is automatically considered superadmin.
I assume that's the dedicated server console?